tests/kani/Vectors/any/sorting.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Sort an arbitrary Vec<u32> of length 3. Assert that the sorting worked.
+#[kani::proof]
+#[kani::unwind(10)]
+fn main() {
+    let mut v: Vec<u32> = kani::vec::any_vec::<3, _>();
+    kani::assume(v.len() == 3);
```

Comment:
Why do you need the assume here?

Reply:
P.S. `>= 3` is probably better, fixing to that.

Reply:
Due to above performance issue, this test case was shrunk to size 2 to avoid timeout. `==2` is required for indexing, so that is `==` is kept.